docker-images/kafka/kafka-thirdparty-libs/2.7.x/pom.xml

```diff
@@ -29,6 +29,8 @@
         <jaeger-client.version>1.6.0</jaeger-client.version>
         <opentracing.version>0.33.0</opentracing.version>
         <opentracing-kafka.version>0.1.15</opentracing-kafka.version>
+        <opentelemetry.version>1.5.0</opentelemetry.version>
+        <opentelemetry-autoconfigure.version>1.5.0-alpha</opentelemetry-autoconfigure.version>
```

Comment:
Are we expected to use `alpha` version?

Reply:
We can start with this, and upgrade as we go?
(I'm sure a decent version will be available before we do next release with this included ...)